indent/verilog_systemverilog.vim

```diff
@@ -169,12 +173,9 @@ function GetVerilog_SystemVerilogIndent()
   " or if it starts with a `define call, which might not require the ; end
   elseif (
     \ last_line =~ ';' . vlog_comment . '*$' ||
-    \ last_line =~ '^\s*`\k\+'
-    \ ) &&
-    \ last_line2 =~ '\<\(`\@<!if\|`\@<!else\|for\|while\|always\|initial\|do\|foreach\|final\)\>\(\s*(.*)\)\?\s*' . vlog_comment . '*$' &&
-    \ last_line2 !~ '\(//\|/\*\).*\<\(`\@<!if\|`\@<!else\|for\|while\|always\|initial\|do\|foreach\|final\)\>' &&
```

Comment:
Just realised the script is already broken for the situations I mentioned. Indenting does not work for the line following ```end else```. Because of this I can't think of many sensible situations that go as ```code // always``` that I can add to the test.